keymanapp/keyman
# chore(android,windows): Update crowdin strings for bwr-NG

Follow-on to #6529 

Fixes Bura-Pabir country codes from "bwr-NT" to "bwr-NG" (Affects Android and Windows)
(edit): Turns out I had misconfigured the custom locale on crowdin. It's now fixed to "bwr-NG", so we don't need the changes to crowdin.yml.

macOS strings not translated on crowdin for bwr-NG.

## User Testing
* **TEST_ANDROID**
1. Load the PR build of Keyman for Android
2. From the Keyman "Settings" menu --> **Display Language** -->
3. Change the display language to "Bura-Pabir"
4. When Keyman reloads, verify the UI is in Bura-Pabir

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. From Keyman Configuration, change the **Display in:** language to "Bura-Pabir"
3. Verify the UI is in Bura-Pabir
